# This is a message to hold data with position and trajectory information
float64 s
float64 ey
float64 epsi
float64 v
float64 x
float64 y
float64 v_x
float64 v_y
float64 yaw
float64 yaw_rate
float64 a_x
float64 a_y
float64 u_a
float64 u_df
float64[] pkg_loss
